kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -1867,6 +1961,131 @@ components:
       required:
         - name
 
+    TopicAnalyzeState:
+      type: object
+      properties:
+        topicName:
+          type: string
+        inProgress:
```

Comment:
when looking at the JSON it makes me feel this is a boolean not an entity. Can we rename it please to make it more transparent?

Reply:
added comment for now, pls let me know if there better solutions

Reply:
`inProgressAnalysisState` & `completedAnalysisState` are okay I believe